packages/governance/test/unitTests/test-singlePluralityBallotCount.js

```diff
@@ -0,0 +1,276 @@
+  await E(creatorFacet).submitVote(tedSeat, [positions[1]], 27n);
+  closeFacet.closeVoting();
+
+  const outcome = await E(publicFacet).getOutcome();
+  t.deepEqual(outcome, BEEF);
+
+  await eventLoopIteration();
+  t.like(storageRoot.getBody('mockChainStorageRoot'), {
+    outcome: 'win',
+    position: BEEF,
+  });
+});
+
+test('single plurality tiecome', async t => {
+  const questionSpec = coerceQuestionSpec({
+    method: ChoiceMethod.PLURALITY,
+    issue: SIMPLE_ISSUE,
+    positions: [FISH, BEEF, PORK],
+    electionType: ElectionType.SURVEY,
+    maxChoices: 1,
+    maxWinners: 1,
+    closingRule: FAKE_CLOSING_RULE,
+    quorumRule: QuorumRule.MAJORITY,
+    tieOutcome: BEEF,
+  });
+  const { publisher, storageRoot } = makePublisherFromFakes();
+  const { publicFacet, creatorFacet, closeFacet } =
+    makeSinglePluralityVoteCounter(
+      questionSpec,
+      0n,
+      FAKE_COUNTER_INSTANCE,
+      publisher,
+    );
+
+  const template = publicFacet.getQuestion();
+  const aliceSeat = makeHandle('Voter');
+  const bobSeat = makeHandle('Voter');
+  const tedSeat = makeHandle('Voter');
+
+  const positions = template.getDetails().positions;
+  t.deepEqual(positions.length, 3);
+
+  E(creatorFacet).submitVote(aliceSeat, [positions[0]], 2n);
+  E(creatorFacet).submitVote(bobSeat, [positions[2]], 27n);
+  await E(creatorFacet).submitVote(tedSeat, [positions[1]], 27n);
+  closeFacet.closeVoting();
+
+  const outcome = await E(publicFacet).getOutcome();
+  t.deepEqual(outcome, [BEEF, PORK]);
+
+  await eventLoopIteration();
+  t.like(storageRoot.getBody('mockChainStorageRoot'), {
+    outcome: 'win',
+    position: [BEEF, PORK],
+  });
+});
+
+test('single plurality revote', async t => {
+  const questionSpec = coerceQuestionSpec({
+    method: ChoiceMethod.PLURALITY,
+    issue: SIMPLE_ISSUE,
+    positions: [FISH, BEEF, PORK],
+    electionType: ElectionType.SURVEY,
+    maxChoices: 1,
+    maxWinners: 1,
+    closingRule: FAKE_CLOSING_RULE,
+    quorumRule: QuorumRule.NO_QUORUM,
+    tieOutcome: BEEF,
+  });
+
+  const { publisher, storageRoot } = makePublisherFromFakes();
+  const { publicFacet, creatorFacet, closeFacet } =
+    makeSinglePluralityVoteCounter(
+      questionSpec,
+      0n,
+      FAKE_COUNTER_INSTANCE,
+      publisher,
+    );
+  const template = publicFacet.getQuestion();
+  const aliceSeat = makeHandle('Voter');
+  const bobSeat = makeHandle('Voter');
+
+  const positions = template.getDetails().positions;
+
+  E(creatorFacet).submitVote(aliceSeat, [positions[0]], 23n);
+  E(creatorFacet).submitVote(bobSeat, [positions[1]], 15n);
+  await E(creatorFacet).submitVote(bobSeat, [positions[2]], 47n);
+  closeFacet.closeVoting();
+
+  const outcome = await E(publicFacet).getOutcome();
+  t.deepEqual(outcome, PORK);
+
+  await eventLoopIteration();
+  t.like(storageRoot.getBody('mockChainStorageRoot'), {
+    outcome: 'win',
+    position: PORK,
+  });
+});
+
+test('single plurality no votes', async t => {
```

Comment:
This will be redundant when single- and multi-plurality are merged. It's fine to drop it.